presto-main/src/main/resources/com/facebook/presto/server/plan.html

```diff
@@ -181,6 +181,7 @@
                 ) +
                 "<div>Memory: " + stats.totalMemoryReservation + "</div>" +
                 "<div>Splits: Q:" + stats.queuedDrivers + ", R:" + stats.runningDrivers + ", F:" + stats.completedDrivers + "</div>" +
+                "<div>Spilled: " + stats.spilledDataSize + "</div>" +
```

Comment:
What do you suggest then? I think not displaying this at all is not a good option either.